### PR TITLE
fixed Codacy code style issues

### PR DESCRIPTION
**Pull request by plata**

### CODE REVIEW

File: phoenicis-multithreading/src/test/java/org/phoenicis/multithreading/ControlledThreadPoolExecutorServiceTest.java [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
 public class ControlledThreadPoolExecutorServiceTest {
     @Test
-    public void testSubmitTask_taskAreExecuted() throws InterruptedException {
+    public void testSubmitTaskTaskAreExecuted() throws InterruptedException {
```
Comment (madoar) [inline phoenicis-multithreading/src/test/java/org/phoenicis/multithreading/ControlledThreadPoolExecutorServiceTest.java]: `...TaskTask...` sounds kind of bad

File: phoenicis-repository/src/test/java/org/phoenicis/repository/types/BackgroundRepositoryTest.java [hunk 2]
```diff
@@ -40,7 +40,7 @@
             .build();
 
     @Test
-    public void testFetchInstallableApplications_taskIsPassed() {
+    public void testFetchInstallableApplicationsTaskIsPassed() {
```
Comment (madoar) [inline phoenicis-repository/src/test/java/org/phoenicis/repository/types/BackgroundRepositoryTest.java]: The method name sounds kind of bad.

File: phoenicis-multithreading/src/test/java/org/phoenicis/multithreading/ControlledThreadPoolExecutorServiceTest.java (hunk 1 above)
Comment (plata) [reply phoenicis-multithreading/src/test/java/org/phoenicis/multithreading/ControlledThreadPoolExecutorServiceTest.java]: Any proposal?

File: phoenicis-repository/src/test/java/org/phoenicis/repository/types/BackgroundRepositoryTest.java (hunk 2 above)
Comment (plata) [reply phoenicis-repository/src/test/java/org/phoenicis/repository/types/BackgroundRepositoryTest.java]: Any proposal?

**Comment by plata** [comment]
I just replaced the "_" in all the methods because Codacy complains about it. I don't really want to think about the names in this pr.

**Comment by madoar** [comment]
I guess we can merge this, but we should open a new Issue about checking the test function names.

**Comment by plata** [comment]
Feel free to do so. To be honest, I don't really care because it's only in the unit tests.

**Pull request merged**